featuretools/primitives/standard/transform/time_series/box_cox.py

```diff
@@ -0,0 +1,51 @@
+from scipy import stats
+from woodwork.column_schema import ColumnSchema
+from woodwork.logical_types import Double
+
+from featuretools.primitives.base import TransformPrimitive
+
+
+class BoxCox(TransformPrimitive):
+    """Applies a transformation to help your data resemble a normal distribution,
+        commonly known as the Box Cox transformation. The input data must be positive.
+        It cannot contain zeroes or negative numbers.
+        Commonly used with time series data, to reduce the variance.
+
+    Args:
+        lmbda (int, float, optional): The exponent to use for the transformation.
```

Comment:
If we leave this as an optional parameter, I think we should describe what happens in the case where it is not specified.